src/Diagnostics.RuntimeHost/Services/DevOpsClient/DevOpsClient.cs

```diff
@@ -112,6 +116,13 @@ public async Task<object> GetFileContentAsync(string filePathInRepo, string reso
 
             try
             {
+                GitRepository repositoryAsync = await gitClient.GetRepositoryAsync(_project, _repoID, (object)null, new CancellationToken());
+                // If branch is not given, use the default of the repo.
+                if (string.IsNullOrWhiteSpace(branch))
+                {
+                    branch = repositoryAsync.DefaultBranch.Replace("refs/heads/", "");
+                }
```

Comment:
Is this part needed? It uses the default branch when version isn't specified by default.

Reply:
Good catch. I missed that. I will change it